plugins/chinesehoroscope.py

```diff
@@ -0,0 +1,94 @@
+# Plugin by Infinity - <https://github.com/infinitylabs/UguuBot>
+
+import requests
+from bs4 import BeautifulSoup
+from sqlalchemy import Table, String, Column, select
+
+from cloudbot import hook
+from cloudbot.util import database
+
+table = Table(
+    'horoscope',
```

Comment:
Change the name of this table, otherwise it conflicts with the existing horoscope table in the horoscope.py plugin